src/BlazorWebView/src/Maui/BlazorWebView.cs

```diff
@@ -11,44 +11,7 @@ namespace Microsoft.AspNetCore.Components.WebView.Maui
 	/// </summary>
 	public partial class BlazorWebView : View, IBlazorWebView
 	{
-		internal static string AppHostAddress { get; } = GetAppHostAddress();
-
-		private const string AppHostAddressAlways0000Switch = "BlazorWebView.AppHostAddressAlways0000";
-
-		private static bool IsAppHostAddressAlways0000Enabled =>
-			AppContext.TryGetSwitch(AppHostAddressAlways0000Switch, out var enabled) && enabled;
-
-		private static string GetAppHostAddress()
-		{
```

Comment:
This logic was moved to a shared source file because so that it could be utilized in non-MAUI scenarios (e.g., WPF).